### ISSUE TITLE
Woo Express: Receiving a "trial expiring" email as soon as I signed up

### ISSUE THREAD

**Issue by markbiek** [Automattic/wp-calypso#84248]
I created a new Woo Express site by visiting `/setup/wooexpress`. 

After the site setup was complete, I received both the "Congratulations" signup email _and_ the "Your free trial ends tomorrow" email, even though I obviously still have 15 days left on the trial.

![CleanShot 2023-11-16 at 08 10 50@2x](https://github.com/Automattic/wp-calypso/assets/917632/01ca9f91-b536-49bd-afc5-0b2d0236ec20)
![CleanShot 2023-11-16 at 08 11 17@2x](https://github.com/Automattic/wp-calypso/assets/917632/5e952706-9820-46db-b570-a5efadfc0fca)

**Comment by markbiek** [comment]
cc @Automattic/serenity (in case anyone knows someone else we should ping on this one).

**Comment by tckasa** [comment]
cc: @bubel and @Automattic/dinobots, for visibility.

FYI, @garethallison

**Comment by bubel** [comment]
@markbiek were you sandboxed while testing this? There is some code that modifies the scheduling of emails when sandboxed.

**Comment by markbiek** [comment]
> @markbiek were you sandboxed while testing this? There is some code that modifies the scheduling of emails when sandboxed.

I'm honestly not sure! Let me try again when I'm confident I'm unsandboxed.

**Comment by markbiek** [comment]
@bubel Yup, that was it. Unsandboxed, I only get the single "Congratulations" email.

Sorry for the false alarm and thanks for the TIL!
